ripley/api/auth_controller.py

```diff
@@ -0,0 +1,109 @@
+"""
+Copyright ©2022. The Regents of the University of California (Regents). All Rights Reserved.
```

Comment:
@mlincoln Copyright year should be 2023 on all files.

Reply:
@lyttam @mlincoln - I created https://jira-secure.berkeley.edu/browse/RIP-58 to update copyright year. All other tools need the same treatment, for which I created more Jiras. 😄

Reply:
Thanks @johncrossman!